java/client/src/org/openqa/selenium/chrome/ChromeDriver.java

```diff
@@ -163,4 +165,25 @@ public ChromeDriver(ChromeDriverService service, Capabilities capabilities) {
     super(new ChromiumDriverCommandExecutor(service), capabilities, ChromeOptions.CAPABILITY);
   }
 
+  public String getCastSinks() {
```

Comment:
Is this specific to Chrome, or to all Chromium-based drivers? I suspect the latter, and if this is the case, please lift these methods up to `ChromiumDriver`

Reply:
I confirmed that it's applicable to Chromium-based drivers (depending on browser support, which varies between browsers and platforms). Moved to these changes to `ChromiumDriver`.